sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/HttpFaultInjectingTests.java

```diff
@@ -138,7 +140,7 @@ public void downloadToFileWithFaultInjection() throws IOException, InterruptedEx
                     null, Context.NONE);
                 byte[] actualFileBytes = Files.readAllBytes(it.toPath());
                 TestUtils.assertArraysEqual(realFileBytes, actualFileBytes);
-                successCount.incrementAndGet();
+                LOGGER.atVerbose().log(() -> "Successful complete count: " + successCount.incrementAndGet());
```

Comment:
nit ;)
```suggestion
                LOGGER.atVerbose()
                .addKeyValue("successCount", successCount.incrementAndGet())
                .log("Download complete");
```

Reply:
why not?

doesn't really matter in tests, but want to check if I miss something